docs/get-started.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Get Started with Xamarin Community Toolkit"
+author: AlexHedley
+description: "Xamarin Community Toolkit provides a single cross-platform API that works with any iOS, Android, or UWP application that can be accessed from shared code no matter how the user interface is created."
+---
+
+# Get Started with Xamarin.CommunityToolkit
+
+Xamarin.CommunityToolkit provides a single cross-platform API that works with any iOS, Android, or UWP application that can be accessed from shared code no matter how the user interface is created. <!--See the [platform & feature support guide](platform-feature-support.md) for more information on supported operating systems.-->
+
+## Installation
+
+Xamarin.CommunityToolkit is available as a NuGet package and is included in every new project in Visual Studio. It can also be added to any existing using Visual Studio with the follow steps.
+
+1. Download and install [Visual Studio](https://visualstudio.microsoft.com/) with the [Visual Studio tools for Xamarin](https://docs.microsoft.com/xamarin/get-started/installation/).
```

Comment:
You should use a site-relative link here (tried to find a reference for this that's open to the outside world, and I can't). So:

https://docs.microsoft.com/xamarin/get-started/installation/

Will become:

/xamarin/get-started/installation/

Basically, links to other docs on docs.microsoft.com should never have https://docs.microsoft.com/ in them.

From an internal page:

Don't append the file extension .md on a link to an article in another repo or docset.

Site-relative links aren't validated at build time like file-relative links. If you have a typo in your link, no warning is generated. Site-relative links are also not click-navigable on the staging server review.docs.microsoft.com. To verify that the link works, replace the branch name in the link's generated URL with master.